lisa/sut_orchestrator/azure/common.py

```diff
@@ -493,6 +496,12 @@ class AzureArmParameter:
     data_disks: List[DataDiskSchema] = field(default_factory=list)
     use_availability_sets: bool = False
     vm_tags: Dict[str, Any] = field(default_factory=dict)
+    
+    virtual_network_resource_group: str = ""
+    virtual_network_name: str = AZURE_NEW_VIRTUAL_NETWORK
+    subnet_prefix: str = AZURE_NEW_SUBNET_PREFIX
+    use_existing_virtual_network: bool = False
```

Comment:
Is the bool value needed?

Reply:
i thought the value was needed.

this seems to work:
```
class WithValue:
    withValue: bool = False

    def Print(self):
        print(self.withValue)
    
WithValue().Print()
```

this complains that `AttributeError: 'WithoutValue' object has no attribute 'withoutValue'`
```
class WithoutValue:
    withoutValue: bool

    def Print(self):
        print(self.withoutValue)

WithoutValue().Print()
```

Reply:
but am happy to be corrected :)

Reply:
I mean, the bool is used to check if the virtual_network_resource_group is set or not. But it can be checked by the virtual_network_resource_group is empty or not. So I think the additional bool value is not necessary.

Reply:
I want to make both runbook and arm template less parameters. They are many values already.

Reply:
ok!

Reply:
doen